### ISSUE TITLE
[en] can take actions → can act

### ISSUE THREAD

**Issue by marcoagpinto** [languagetool-org/languagetool#6516]
Hello!

Is this a good idea for a rule?:
`and you can take actions against the entities.`

Suggest?:
`and you can act against the entities.`

**Comment by babycaseny** [comment]
I don't think it is grammatical but something more like usage preferences. You can see “take action against ……” on legal environments. Changing them to “act against ……” will have the original feeling lost.

**Comment by Nomad-Q** [comment]
Hello, I have added a commit. After adding a new rule to detect the word "take action", it will give out a suggestion like this
<img width="593" alt="截屏2022-04-24 上午10 51 38" src="https://user-images.githubusercontent.com/74593761/164954054-c94bb157-3429-4a20-ad8f-097a0f29639a.png">
:
